proxstar/vm.py

```diff
@@ -250,8 +250,8 @@ def create_vm(proxmox, user, name, cores, memory, disk, iso):
         name=name,
         cores=cores,
         memory=memory,
-        storage='ceph',
-        virtio0='ceph:{}'.format(disk),
+        storage=app.config['PROXMOX_VM_STORAGE'],
+        virtio0='{}:{}'.format(app.config['PROXMOX_VM_STORAGE'], disk),
```

Comment:
I may have forgotten to include this env variable in the `config.py`. Whoops.

Reply:
f-strings? :eyes: